docs/docs-backend/Autorização e permissões.md

```
@@ -0,0 +1,119 @@
+9. Adicione testes para o catálogo, o guard, a camada de autorização e os resolvers ou controllers alterados.
+10. Atualize stories quando a nova permissão afetar a UI de concessões ou permissões.
+
+## Cuidados
+
+Não trate checks de frontend como autorização.
+
+Não adicione novos papéis de Keycloak para permissões de negócio sem mudar a arquitetura conscientemente.
+
+Não conceda `event-manager#super-admin` para resolver falta de escopo em uma operação comum.
+
+Não edite permissões antigas em migrações já aplicadas. Quando a mudança exigir alterações no banco de dados, crie uma nova migração.
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)